tests/trestle/core/base_model_test.py

```diff
@@ -254,6 +258,21 @@ def test_copy_to() -> None:
     _ = remark.copy_to(common.RiskStatus)
 
 
+def test_copy_components() -> None:
+    """Test copying across similar but different objects."""
+    state_obj = ssp.State1('under-development')
+    sys_component = ssp.SystemComponent(
+        uuid='A0000000-0000-4000-8000-000000000000',
```

Comment:
ideally SAMPLE_UUID_STR
I see about 10 examples of the A0000... string in the code and data and it would be good to clean them up over time

Reply:
On my version I cannot see any other A000.. string except the constant in *python* files. I've resolved this one so we should be clean now.